charts/rancher-webhook/105.0.1+up0.6.1-rc.1/Chart.yaml

```diff
@@ -8,7 +8,7 @@ annotations:
   catalog.cattle.io/rancher-version: '>= 2.9.0-0 < 2.10.0-0'
   catalog.cattle.io/release-name: rancher-webhook
 apiVersion: v2
-appVersion: 0.6.0-rc.1
+appVersion: 0.6.1-rc.1
 description: ValidatingAdmissionWebhook for Rancher types
 name: rancher-webhook
-version: 105.0.0+up0.6.0-rc.1
+version: 105.0.1+up0.6.1-rc.1
```

Comment:
Can you elaborate on why this is 105.0.1? This part is the rancher compat version, would should still be 105.0.0 for 2.10 AFAIK